### DISCUSSION TITLE
Question about the weights "detectron2://DeepLab/R-52.pkl"

### DISCUSSION THREAD

**Discussion by mvdelt** [bowenc0221/panoptic-deeplab#83] [Q&A] [answered]
Hi,
Both `panoptic_deeplab_R_52_os16_mg124_poly_90k_bs32_crop_512_1024_dsconv.yaml` and
`panoptic_deeplab_R_52_os16_mg124_poly_200k_bs64_crop_640_640_coco_dsconv.yaml`
are using the weights `"detectron2://DeepLab/R-52.pkl"`. 

Where this weights came from? 
Is this a pre-trained weights or just a randomly initialized weights?

Thanks.

**Comment by bowenc0221** [comment, marked as answer]
`detectron2://DeepLab/R-52.pkl` is ImageNet pre-trained weights for a modified ResNet-50 backbone.

**Reply by mvdelt** [reply]
Thank you for your answer.
So, `https://dl.fbaipublicfiles.com/detectron2/PanopticDeepLab/Cityscapes-PanopticSegmentation/panoptic_deeplab_R_52_os16_mg124_poly_90k_bs32_crop_512_1024_dsconv/model_final_23d03a.pkl` is a weights trained with cityscapes dataset based on `detectron2://DeepLab/R-52.pkl`?

I'm trying to train panoptic-deeplab with my custom dataset(x-ray images), so I need to choose weights for baseline.
I guess training from `detectron2://DeepLab/R-52.pkl` would be better.. right?

**Reply by bowenc0221** [reply]
Yes, `model_final_23d03a.pkl` is the cityscapes model.
And I would suggest you start with the `detectron2://DeepLab/R-52.pkl` weight for your custom dataset.
